### ISSUE TITLE
auto_html version requirements holding back rexml upgrade

### ISSUE THREAD

**Issue by KieranP** [dejan/auto_html#195]
https://www.ruby-lang.org/en/news/2024/07/16/dos-rexml-cve-2024-39908/

auto_html (2.1.1) requires rexml (~> 3.2.5)

Which means I cannot upgrade to rexml 3.3.2 or later which includes the bug fixes.

**Comment by ElliottRoche** [comment]
Yeah I'm curious about this too. Causing some issues with security given vulnerabilities in older versions of rexml.

**Comment by dejan** [comment]
New version released https://github.com/dejan/auto_html/releases/tag/v2.2.0
